client/src/components/History/CurrentHistory/HistoryOperations/SelectionStatus.vue

```diff
@@ -1,10 +1,20 @@
 <template>
     <b-button-group size="sm">
-        <b-button v-if="hasSelection" variant="link" data-test-id="clear-btn" @click="resetSelection">
-            <span class="fa fa-fw fa-times" title="Clear selection" />
+        <b-button
+            v-if="!allSelected"
+            variant="link"
+            data-test-id="select-all-btn"
+            title="Select all"
+            @click="selectAll">
+            <span v-if="!hasSelection">Select </span>All
```

Comment:
Thank you for working on this. Here I noticed that this shifts the selection dropdown to the left and back to the right depending on the selection. This is a lot of motion and I am not sure if its necessary. Previously it was steady.

Reply:
You are right, I don't like the shifting either, but I think it's important to be able to select all items again in case you change your mind, without having to cancel the selection completely and then start over.
I'll try moving the `Select all` and `Clear` to the left of the selection dropdown to see if it gets 'better' or we can think of another solution :+1:

Reply:
These changes are no longer part of this PR, we can follow up with the watch enhancements in a different PR.